src/App.js

```diff
@@ -1,3 +1,4 @@
+import { useState } from 'react'
 import './App.css'
```

Comment:
Due to the fact that you are working with styled-components, you will most probably not make use of the App.css and its import.